lib/views/schedule.dart

```diff
@@ -26,6 +27,7 @@ class Schedule extends StatelessWidget {
     return new ListView(
       children: children,
     );
+
```

Comment:
Please also format the code in this file.
 - remove empty lines
 - make spaces like in other files etc

-> Ctrl + Alt + L (should be standard in IntelliJ and Android Studio)